src/components/overlay/index.tsx

```
@@ -0,0 +1,173 @@
+            case 'rect':
+            case 'ellipse':
+            case 'generatedui':
+                return {
+                    x: shape.x,
+                    y: shape.y,
+                    w: shape.w,
+                    h: shape.h, // Use the shape's height which gets updated by the component
+                }
+            case 'freedraw':
+                if (shape.points.length === 0) return { x: 0, y: 0, w: 0, h: 0 }
+                const xs = shape.points.map((p) => p.x)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)